test/commands/where_test.rb

```diff
@@ -82,6 +82,33 @@ def test_where_displays_backtraces_using_short_callstyle
 
       check_output_includes(*expected_output)
     end
+
+    def test_where_displays_instance_exec_block_frames
+      enter 'where'
+      program = strip_line_numbers <<-EOP
+         1:  module Byebug
+         2:    class #{example_full_class}
+         3:      def foo
+         4:        Object.new.instance_exec do
+         5:          byebug
+         6:        end
+         7:      end
+         8:     end
```

Comment:
Just a super small style nitpick, can you add an empty line between the class and the method call and update the test accordingly?